profiles/common/modules/features/cce_basic_config/cce_basic_config.install

```diff
@@ -474,21 +474,63 @@ function cce_basic_config_install() {
 }
 
 /**
- * Initialize 'theme default' and 'admin theme'.
+ * Form for the installation profile task "Choose theme".
+ *
+ * This form is shared by both standard and communities profile, hence it is
+ * exposed by the 'cce_basic_config' module.
+ *
+ * @see multisite_drupal_standard_install_tasks()
+ * @see multisite_drupal_communities_install_tasks()
  */
-function _cce_basic_config_post_install_theme() {
-  // Define themes.
-  $themes = array(
-    'theme_default' => 'europa',
-    'admin_theme' => 'seven',
-  );
-  theme_enable($themes);
-  $themes[] = array('dependency' => 'atomium');
-  foreach ($themes as $var => $theme) {
-    if (!is_numeric($var)) {
-      variable_set($var, $theme);
-    }
+function cce_basic_config_profile_theme_default_form() {
+  $form = array();
+  $form['container'] = array(
+    '#type' => 'fieldset',
+    '#title' => st('Choose theme'),
+    '#collapsible' => FALSE,
+    '#tree' => FALSE,
+  );
+  $form['container']['theme_default'] = array(
+    '#type' => 'radios',
+    '#required' => TRUE,
+    '#options' => array(
+      'ec_resp' => st('EC Resp'),
+      'europa' => st('Europa'),
+    ),
+    '#default_value' => 'ec_resp',
+  );
+  $form['container']['submit'] = array(
+    '#type' => 'submit',
+    '#value' => st('Continue'),
+  );
+  return $form;
+}
+
+/**
+ * Form submit.
+ *
+ * @see multisite_drupal_standard_install_tasks()
+ */
+function cce_basic_config_profile_theme_default_form_submit($form, &$form_state) {
+  $theme_default = $form_state['values']['theme_default'];
+  $theme_list = array($theme_default);
+  if ($theme_default == 'europa') {
+    $theme_list[] = 'atomium';
+    module_enable(array('nexteuropa_ecl_migrate'));
```

Comment:
**Improvement for later**: considering the disabling of the module when it after the migration execution.

If we agree, we could see that in a follow-up ticket.